tests/scripts/all-core.sh

```diff
@@ -1,3 +1,4 @@
+#! /usr/bin/env bash
```

Comment:
This file is not executable (it's meant to be sourced) so it doesn't need a shebang line. Was a tool complaining? if so, we should fix the tool.